cluster/gce/windows/k8s-node-setup.psm1

```diff
@@ -386,6 +386,38 @@ function DownloadAndInstall-KubernetesBinaries {
   Remove-Item -Force -Recurse $tmp_dir
 }
 
+# Downloads the csi-proxy binaries from kube-env's CSI_PROXY_TAR_URL and
+# puts them in a subdirectory of $env:K8S_DIR.
+#
+# Required ${kube_env} keys:
+#   CSI_PROXY_TAR_URL
+function DownloadAndInstall-CSIProxyBinaries {
+
+  $tmp_dir = 'C:\k8s_tmp'
+  New-Item -Force -ItemType 'directory' $tmp_dir | Out-Null
+
+
+  if (Test-IsTestCluster $kube_env) {
+
+    if (ShouldWrite-File ${env:NODE_DIR}\csi-proxy.exe) {
+      # Here we use $env to set up url first, later will change to use kube_env.
+      $urls = ${env.CSI_PROXY_TAR_URL}
```

Comment:
Does `env.` work? We use `env:` everywhere else in the script...

But anyway, since this variable is only used once, I think you can just use CSI_PROXY_TAR_URL directly from `$kube_env` - no need to insert it into `$env` in `Set-EnvironmentVars`.